server/sonar-web/src/main/js/apps/component-measures/drilldown/ComponentCell.js

```diff
@@ -65,23 +65,24 @@ export default class ComponentCell extends React.PureComponent {
   }
 
   render() {
-    const { branch, component } = this.props;
+    const { branchLike, component } = this.props;
     return (
       <td className="measure-details-component-cell">
         <div className="text-ellipsis">
+          {/* TODO make this <a> link a react-router <Link /> */}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([javascript:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fbranch%2Ffirst-class-pull-request&issues=AWFW-hUEsS4rYBZqFcKM&open=AWFW-hUEsS4rYBZqFcKM)

[ISSUE_KEY]: AWFW-hUEsS4rYBZqFcKM